test/test.dart

```diff
@@ -329,13 +369,23 @@ void testAppGoldens(
             _testersPlayerInterfaceColorStyle.remove(tester);
           });
           EditableText.debugDeterministicCursor = true;
-          ThemeControl.instance.setThemeLightMode(lightTheme);
-          Settings.playerInterfaceColorStyle.set(playerInterfaceColorStyle);
           _testersLightTheme[tester] = lightTheme;
           if (nonDefaultPlayerInterfaceColorStyle) {
             _testersPlayerInterfaceColorStyle[tester] = playerInterfaceColorStyle;
           }
-          return await test(tester);
+          return tester.runAppTest(
+            initialization: () => initialization?.call(),
+            postInitialization: () {
+              ThemeControl.instance.setThemeLightMode(lightTheme);
+              Settings.playerInterfaceColorStyle.set(playerInterfaceColorStyle);
+              postInitialization?.call();
+            },
+            () => test(tester),
+            goldenCaptureCallback: () => tester.screenMatchesGolden(
+              Invoker.current!.liveTest.test.name.split(' | theme')[0].replaceAll(' ', '.'),
```

Comment:
Filename is already being built here https://github.com/nt4f04uNd/sweyer/blob/master/test/test.dart#L259

I'm don't complete understand the purpose of this code, but it seems like using the `description` parameter would already would already enrich it with the required parameters, maybe except from the group name

This code just seems to be not very scalable